pages/linux/mopac.md

```diff
@@ -0,0 +1,16 @@
+# mopac
+
+> MOPAC (Molecular Orbital PACkage) is a semiempirical quantum chemistry program based on Dewar and Thiel's NDDO approximation.
+> More information: <http://openmopac.net/>.
```

Comment:
This website seems to be not updated, and it doesn't support HTTPS.
Maybe link to the GitHub repo:
```suggestion
> More information: <https://github.com/openmopac/mopac>.
```
Or link to the new documentation site, but it's still under development:
```suggestion
> More information: <https://openmopac.github.io/>.
```